catalog.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>NЁRDS</title>
+</head>
+<body>
+	<header>
+		<nav>
+			<a href="index.html">
+				<img src="https://via.placeholder.com/150x100" alt="">
+			</a>
+			<ul>
+				<li><a href="studio.html">Студия</a></li>
+				<li><a href="clients.html">Клиенты</a></li>
+				<li><a>Магазин</a></li>
+				<li><a href="contacts.html">Контакты</a></li>
+			</ul>
+			<div>
+				<a href="blank.html">Корзина</a>
+			</div>	
+		</nav>
+	</header>
+	<main>
+		<h1>Магазин готовых шаблонов</h1>
+		<section>
+			<h2>Фильтр</h2>
+			 <!-- Здесь будет форма фильтра -->
+		</section>
+		<section>
+			<h2>Форма сортировки</h2>
+			<ul>
+				<li><a>по цене</a></li>
+				<li><a href="">по типу</a></li>
+				<li><a href="">по назначению</a></li>
+			</ul>
+			<ul>
+				<li><a>по убыванию</a></li>
+				<li><a href="">по возрастанию</a></li>
+			</ul>
+		</section>
+		<section>
+			<h2>Товары</h2>
+			<ul>
+				<li>
+					<h3>Sedona</h3>
```

Comment:
Нужно текст обернуть ссылкой. Вот так:
```suggestion
					<h3><a href="#">Sedona</a></h3>
```

То, что тут должна быть ссылка есть указание в [ТЗ](https://up.htmlacademy.ru/htmlcss/29/project/nerds#nerds-3-10).